File: GoLang/6 kyu/Array.diff/main.go
package main

import "fmt"

func notInArray(array []int, elem int) bool {
	for _, value := range array {
		if value == elem {
			return false
		}
	}
	return true
}

func ArrayDiff(a, b []int) []int {
	var result []int
	for _, elem := range a {
		if notInArray(b, elem) {
			result = append(result, elem)
		}
	}
	return result
}

func main() {
	fmt.Println(ArrayDiff([]int{1, 2, 2, 4, 5, 5, 5, 1, 7, 0, 9, 0, 2, 3}, []int{2, 5, 9}))
}

File: Python/6 kyu/Array.diff/main.py
def array_diff(a, b):
    for elem in b:
        while elem in a: a.remove(elem)
    return a

print(array_diff([1, 2, 2, 4, 5, 5, 5, 1, 7, 0, 9, 0, 2, 3],[2, 5, 9]))